### PR TITLE
MSO web dev template using Wagtail in Django

### PR DESCRIPTION
**Pull request by KWaloen**
Here is the template that we can use to develop the website for the MSO project.
The template includes a title field, a hero text and body created in the model and migrated to the database.
These fields can be edited in the localhost:8000/admin panel and since we use rich text in the models mentioned above, we (and the client) can edit the text easily and format it as we like. 
Documentation as been added in the model and in the README for instructions on how to use this template.

TODO: 
-Check the structure of this webpage to make sure it makes sense and there is no redundancy.
-Look into and understand how CSS works here.

### CODE REVIEW

**Comment by skivdal** [comment]
Very nice! Looks good
I wanna test it out on my laptop tomorrow, then probably merge it :D

File: home/models.py [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+from wagtail.admin.panels import MultiFieldPanel, FieldPanel
+from wagtail.fields import RichTextField
 
 from wagtail.models import Page
 
+from wagtail.models import Page
+from wagtail.fields import RichTextField
+
+# import MultiFieldPanel:
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel
```
Comment (skivdal) [inline home/models.py:11]: Some of these imports are redundant

I think PyCharm can fix that automatically, try `Ctrl`+`Alt`+`Shift`+`L`, or double tapping `Shift` and searching for "Reformat Code" or similar

File: home/templates/home/home_page.html [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
 
 {% block body_class %}template-homepage{% endblock %}
 
-{% block extra_css %}
-
-{% comment %}
-Delete the line below if you're just getting started and want to remove the welcome screen!
-{% endcomment %}
-<link rel="stylesheet" href="{% static 'css/welcome_page.css' %}">
-{% endblock extra_css %}
```
Comment (skivdal) [inline home/templates/home/home_page.html:12]: The  `{% block extra_css %}` and `{% endblock extra_css %}` should probably stay, as this allows you to inject additional CSS to the page header specifically for this template
But the lines in between those should be removed :D

File: mso_website/urls.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
     path("django-admin/", admin.site.urls),
     path("admin/", include(wagtailadmin_urls)),
     path("documents/", include(wagtaildocs_urls)),
+    path("homepage/", include(wagtaildocs_urls)),
```
Comment (skivdal) [inline mso_website/urls.py:15]: I don't know what this :sweat_smile:

File: README.md [hunk 4]
```diff
@@ -72,8 +72,17 @@ The project depends on a SQLite database. This is not yet initialized and is the
 
 The website should now be in working order on your computer. On running the website with the Play button in PyCharm (or command if not using PyCharm), you should no longer see an error message.
 
+### Creating new pages.
+
+New pages can be created by going to localhost:8000/admin -> Pages -> click on the pencil next to the title -> click on the triple dots on the top of the page -> Add child page
+You can navigate to the page that you created by using the title of your page like this: localhost:8000/PAGE_TITLE after you publish using the drop up menu at the bottom of the page.
```
Comment (skivdal) [inline README.md:78]: If you wrap these links like this: `[localhost:8000/admin](http://localhost:8000/admin)`, they will highlight and become clickable when reading the document

The syntax is: `[friendly name](actual URL)`

**Review by skivdal — COMMENTED** [review]
Some suggestions for minor tweaks that can improve the code :D

File: mso_website/urls.py (hunk 3 above)
Comment (KWaloen) [reply mso_website/urls.py:15]: Brainfart, me neither 😓

File: README.md (hunk 4 above)
Comment (KWaloen) [reply README.md:78]: Nice, thanks for the info, I'll incorporate this 🙌

File: home/models.py (hunk 1 above)
Comment (KWaloen) [reply home/models.py:3]: Didnt work for me but I manually removed unused imports.

**Comment by skivdal** [comment]
LETSGOOO :star_struck:

**Pull request merged**